gooddata_agents.py

```diff
@@ -29,6 +29,7 @@ class GoodDataAgentsDemo:
     def __init__(self) -> None:
         self.args = self.parse_arguments()
         load_dotenv()
+        OpenAI.api_key = os.getenv("OPENAI_API_KEY")
```

Comment:
Does it have to be in the constructor? Can we make it global?

Reply:
I have it here because of the load_dotenv(), but sure, we might want to do it globally